pkgs/development/python-modules/flask-babel/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, buildPythonPackage
+, python
+, fetchPypi
+, flask
+, Babel
+, jinja2
+, pytz
+, speaklater
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-Babel";
+  version = "0.11.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ff9n165vhf1nhv6807ckhpp224jw7k7sd7jz5kfh3sbpl85gmy0";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    Babel
+    jinja2
+    pytz
+    speaklater
+  ];
+
+  checkPhase = ''
+    ${python.interpreter} -m unittest discover -s tests
+  '';
+
+  meta = with stdenv.lib; {
+    description = "i18n and l10n support for Flask based on Babel and pytz";
```

Comment:
Just use

> Adds i18n/l10n support to Flask applications

from PyPI and you get the capitalization required by https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.